docs/reference/indices/get-index.asciidoc

```diff
@@ -15,3 +15,64 @@ alias or wildcard expression is required.
 
 The get index API can also be applied to more than one index, or on
 all indices by using `_all` or `*` as index.
+
+[float]
+=== Skipping types
+
+Types are scheduled to be fully removed in Elasticsearch 8.0 and will not appear
```

Comment:
I think it would be nice to mention that the type will not be returned in 7.0 by default. This is also what we do in other 6.x documentation: https://github.com/elastic/elasticsearch/commit/cd904337c6651578ab13ee4b1c74bdacb7b1a061#diff-0ee8c56315540fb266357d5163321ecf

Reply:
I'll just take that text-blob then